src/randoop/operation/MethodParser.java

```diff
@@ -0,0 +1,51 @@
+package randoop.operation;
+
+import java.lang.reflect.Method;
+
+import randoop.types.TypeNames;
+
+/**
+ * MethodParser provides methods to recognize and extract methods from
+ * string representations of their signature.
+ * 
+ * @author bjkeller
+ *
+ */
+public class MethodParser {
+
+  public static Method getMethodForSignature(String signature) throws OperationParseException {
```

Comment:
Don't forget the documentation.  For example, what is the format of the parameter?  Java has 3 or 4 different representations for signatures.